futures-channel/src/mpsc/atomic_waker.rs

```diff
@@ -0,0 +1 @@
+include!("../../../futures-util/src/task/atomic_waker.rs");
```

Comment:
I'm wondering if it would be better to export `atomic_waker` from `futures_core` privately (via a `#[doc(hidden)] mod __internal { ... }`) or something so that the code can be shared. WDYT?

Reply:
I think the best way to solve this issue would be using not yet implemented [RFC 1977](https://github.com/rust-lang/rust/issues/44663).

But since this RFC won't be implemented any time soon, I'll update the PR with `AtomicWaker` moved to `futures-core`.